package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "oak-ai-web-app",
+  "name": "oak-ai-lesson-assistant",
```

Comment:
I've noticed this in a few places. Have we decided that labs dot will no longer have other apps and experiments on it and Aila is the only thing that this domain will deal with? 

Eg. we have the quiz designer.

The original plan was labs dot was a place for several apps and Aila was one of them. This would move us to just having Aila at this subdomain?